backend/app/Services/Erp/MaterialLotImportService.php

```
@@ -0,0 +1,202 @@
+
+            if ($existing) {
+                // Never shrink quantity_received below what is available — the ERP
+                // may only report the free remainder of a larger receipt.
+                $attributes['quantity_received'] = max((float) $existing->quantity_received, $quantity);
+                $existing->update($attributes);
+                $this->syncLotBalance($existing, $warehouse, $quantity);
+
+                return $this->updated();
+            }
+
+            $lot = MaterialLot::create([
```

PHPStan (2.2.6)

[error] Call to an undefined static method App\Models\MaterialLot::create().


(staticMethod.notFound)